rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2222,8 +2462,8 @@ private boolean verifyLockDoesNotSurpassLockingCap(BtcTransaction btcTx, Address
         return false;
     }
 
-    private Optional<ReleaseTransactionBuilder.BuildResult> getRefundingTransaction(BtcTransaction btcTx, Address senderBtcAddress, boolean isFastBridgeCompatible) throws IOException {
-
+    private Optional<ReleaseTransactionBuilder.BuildResult> getRefundingTransaction(
+        BtcTransaction btcTx, Address refundingBtcAddress, boolean isFastBridgeCompatible) throws IOException {
```

Comment:
I think it does not meet the code standards

Reply:
This method is no longer nedeed